packages/music/tsconfig.cjs.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "build/cjs",
+    "module": "CommonJS",
+    "target": "ES2021"
```

Comment:
If you target ES2021, it's only guaranteed to work on Node 16+ (see https://node.green/#ES2021). If you want to support older versions, you can step this down, but be sure to add `ES2021` to the `lib` in either this or the base `tsconfig.json`, or the compiler will balk at your use of things like `.replaceAll`.

Reply:
can you provide a test code for node v14

Reply:
this reason for minimum requirement node v16 is, some imports are not available in node v14, for example `node:events` hence minimum requirement could not changed back to node v16. Also discord.js has also deprecated support for node v14. for the same reason.

Reply:
I couldn't find it documented anywhere that node v16 was the minimum version. I'm fine with whatever. 

> can you provide a test code for node v14

I'm confused here. What is it you'd like from me here?

Reply:
> I couldn't find it documented anywhere that node v16 was the minimum version. I'm fine with whatever.
https://discord-ts.js.org/docs/installation#requirements

> I'm confused here. What is it you'd like from me here?
nvm, but I have released patch. So test again to make sure it's now working with commonjs

Reply:
Oh, I just looked at the README, heh. I should RTFM, right?

Reply:
sorry, what's `RTFM`?

Reply:
if that's mean read the first manual, don't worry I will add this in readme too

Reply:
It's "Read the F!@#ing Manual." It was poking fun at myself, saying I was asking questions that you clearly documented, and you shouldn't have to keep answering them. It was supposed to be a joke (at my expense). 😄

Reply:
https://github.com/oceanroleplay/discord.ts#-requirements

Reply:
Heh, thanks!